index.bs

```diff
@@ -1066,9 +1063,9 @@ The <dfn>power preference</dfn> indicates preference as related to power consump
 <dl>
 <dt>"<code><dfn data-lt="power-preference-default">default</dfn></code>"</dt>
 <dd>Let the user agent select the most suitable behavior.</dd>
-<dt>"<code><dfn data-lt="power-preference-high-performance">high-performance</dfn></code>"</dt>
+<dt>"<code><dfn ignore data-lt="power-preference-high-performance">high-performance</dfn></code>"</dt>
```

Comment:
What's the intention of adding "ignore"? Its usage is missed in the commit log?

Reply:
That's to silence a warning... because with the "validate MLContext" algorithm is removed, the enum member is never actually referenced in the spec! That's because there's an implicit conversion of the enum value to the corresponding `MLPowerPreference` enum value. The proposed follow-on work discussed in #497 would dedupe the enums, and turn this definition into a definition for the `MLPowerPreference` enum, and "ignore" can go away.

I'll update the commit log to capture this.

Reply:
Thanks for the explanation. The updated commit log looks good.